express-hello-world/index.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const app = express();
+
+app.use((req, res) => {
+  res.send('hello');
+});
+
+app.listen(3000, () => {
+});
```

Comment:
Here it is a good idea to put a console log in the listen callback so you know your server started successfully and a reminder of the PORT number.